Assignments/A3/Distro/QLearn.h

```diff
@@ -33,13 +33,21 @@
 #define lambda .5 // Discount rate for future rewards
 #define max_graph_size 32 * 32
 
-#define numFeatures 2 // UPDATE THIS to be the number of features you have
+#define numFeatures 5 // UPDATE THIS to be the number of features you have
 #define numActions 4
 #define BIG_DBL 1000000.0
 #define EPSILON 1e-9
 
 enum direction { UP = 0, RIGHT = 1, DOWN = 2, LEFT = 3 };
 
+// BEGIN STRUCT DEFS
+typedef struct Point
+{
+  double x;
+  double y;
+} Point;
+// END STRUCT DEFS
```

Comment:
Let's avoid using `Point` and stick to `int pos[2]` for now.
1. The struct is only used in one function (aside from `length`) at the moment.
2. The assignment's template code already uses the concept of an `int pos[2]` (or `int poss[...][2]`) - we should stick with this to avoid using multiple "languages" to denote a location on the grid.